.github/workflows/pr.yml

```diff
@@ -0,0 +1,21 @@
+name: Lint and Test
+
+on:
+  pull_request:
+
+jobs:
+  all:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Use Node.js 18.8.x
```

Comment:
here's hoping we can do both major LTS versions
(maybe add a comment about why not yet? https://github.com/Agoric/agoric-sdk/issues/8636 )

Reply:
👍 [fccae72](https://github.com/Agoric/dapp-offer-up/pull/33/commits/fccae72fed0f1a6c5286bd6f587f1b50e4a77480)